wgpu-hal/src/gles/emscripten.rs

```diff
@@ -0,0 +1,22 @@
+extern "C" {
+    /// returns 1 if success. 0 if failure. extension name must be null terminated
+    fn emscripten_webgl_enable_extension(
+        context: std::ffi::c_int,
+        extension: *const std::ffi::c_char,
+    ) -> std::ffi::c_int;
+    fn emscripten_webgl_get_current_context() -> std::ffi::c_int;
+}
+/// if we have debug extension and if we can enable it -> use unmasked vendor/renderer
```

Comment:
```suggestion
/// If we have the debug extension and if we can enable it, enable it. 
```

Reply:
moved this comment to adapter file because that's where we actually make this decision.